cypress/integration/checkin/checkin.basic.spec.js

```diff
@@ -1,11 +1,13 @@
 import generateItemBarcode from '../../support/utils/generateItemBarcode';
 import checkoutActions from '../../support/fragments/checkout/checkout';
+import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
 
 describe('Check In', () => {
   let ITEM_BARCODE;
 
   beforeEach(() => {
     ITEM_BARCODE = generateItemBarcode();
```

Comment:
You can just assign value when you are creating variable

`let ITEM_BARCODE = generateItemBarcode();`